frontend/src/plugins/veeValidate.ts

```diff
@@ -29,7 +30,13 @@ extend('confirmed', {
 extend('min', {
   ...min,
   params: ['length', 'name'],
-  message: 'Your {name} must be at least {length} characters',
+  message: 'Your {name} about yourself must be at least {length} characters',
```

Comment:
https://github.com/BinaryStudioAcademy/bsa-winter-2022-look/blob/develop/frontend/src/bundles/auth/components/SetUpForm.vue#L39
It was already in use, you changed not only one place validation

Reply:
Fixed